### ISSUE TITLE
Loosen provider constraints

### ISSUE THREAD

**Issue by phene** [vmware/terraform-aws-avi-alb-deployment-aws#28]
### Is your feature request related to a problem? Please describe.

The provider constraints are too strict. This module should have no issue with 5.0.0+ versions of the aws provider or newer versions of the null provider

### Describe the solution you'd like

Loosen the provider constraints. e.g.:

```
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = ">= 4.37.0"
    }
    null = {
      source  = "hashicorp/null"
      version = ">= 3.2.0"
    }
  }
```

### Additional context

I may need to run this terraform module with others and want to lock down my provider versions elsewhere. Additionally, I may need to pre-package my providers to avoid fetching them from the internet in restricted environments. I would like to avoid having to package several versions of a provider.

**Comment by jpeters-vmw** [comment]
Fixed in v1.0.12

**Comment by phene** [comment]
Nevermind -- looks like @jpeters-vmw solved this while I was writing this up! https://github.com/vmware/terraform-aws-avi-alb-deployment-aws/commit/4c6c04f5a60b9941289a5c6d68d2a3ba4129bf0e
